ecl/eclagent/eclagent.cpp

```diff
@@ -2502,6 +2502,8 @@ bool EclAgent::checkPersistUptoDate(IRuntimeWorkflowItem & item, const char * lo
         errText.appendf("Rebuilding PERSIST('%s'): Saved CRC isn't present", logicalName);
```

Comment:
Regarding @jakesmith comment on IWUResults constants. This will be done in this jira ticket: https://track.hpccsystems.com/browse/HPCC-14601